poi/src/main/java/org/apache/poi/util/CodepointsUtil.java

```diff
@@ -26,13 +26,21 @@ public class CodepointsUtil {
     /**
      * @param text to iterate over
      * @return iterator with Strings representing the codepoints
-     * @see #primitiveIterator(String) a more performant iterator
      */
-    public static Iterator<String> iteratorFor(String text) {
-        return text.codePoints()
-                .mapToObj(codePoint -> new String(Character.toChars(codePoint)))
-                .iterator();
-    }
+  public static Iterator<String> iteratorFor(String text) {
+    PrimitiveIterator.OfInt iter = primitiveIterator(text);
+    return new Iterator<String>() {
+      @Override
+      public boolean hasNext() {
+        return iter.hasNext();
+      }
+
+      @Override
+      public String next() {
+        return Character.toString(iter.next());
```

Comment:
was this method not added after Java 8? We require Java 8 support.

Reply:
see https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Character.html#toString(int)

only added in Java 11